platform/qt/src/qmapboxgl.cpp

```diff
@@ -735,6 +805,14 @@ QStringList QMapboxGL::getClasses() const
     return classNames;
 }
 
+/*!
+    Sets the \a duration and \a delay of style class transitions. An animation
+    morphing a style property value to another value is performed when a new
+    class is set.
+
+    \deprecated
```

Comment:
As far as I know, [global transition options](https://www.mapbox.com/mapbox-gl-style-spec/#root-transition) are not deprecated and not under consideration for removal.